packages/dashboard/src/utils/utils.ts

```diff
@@ -104,21 +126,20 @@ export const getNetworkName = async (chainId: number) => {
 };
 
 export const getDisplayName = async (
-  library: providers.Web3Provider,
+  provider: providers.BaseProvider,
   address: string
 ) => {
-  const ensName = await reverseLookup(library, address);
+  const ensName = await reverseLookup(provider, address);
```

Comment:
Did you get this working? It seems to only be returning `undefined` for me, and I have reverse lookup setup.